pyquil/api/compiler.py

```diff
@@ -161,3 +166,69 @@ def get_job_fn():
         return wait_for_job(get_job_fn,
                             ping_time if ping_time else self.ping_time,
                             status_time if status_time else self.status_time)
+
+    def _pipo_payload(self, clifford, pauli):
+        """
+        Prepares a JSON payload for conjugating a Pauli by a Clifford - see apply_clifford_to_pauli.
+
+        :param Program clifford: A Program that consists only of Clifford operations.
+        :param PauliTerm pauli: A PauliTerm to be acted on by clifford via conjugation.
+        :return: The JSON payload, with keys "clifford" and "pauli".
+        """
+        max_qubit = max(max(pauli.get_qubits()), max(clifford.get_qubits()))
+        pauli_string = ["I" for _ in range(max_qubit + 1)]
+        for index, term in pauli.operations_as_set():
+            pauli_string[int(index)] = term
```

Comment:
This functionality might prefer to live as a method on `PauliTerm` instead of here

Reply:
Sure.